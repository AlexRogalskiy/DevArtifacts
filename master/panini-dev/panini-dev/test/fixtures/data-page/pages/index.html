---
kittens: Yes, kittens.
---
Page: {{kittens}}
