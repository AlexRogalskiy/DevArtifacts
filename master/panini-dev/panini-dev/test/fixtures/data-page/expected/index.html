<html>
  <body>
    Layout: Yes, kittens.
    Page: Yes, kittens.
    
    Partial: Yes, kittens.
  </body>
</html>
